staging/cse/windows/provisioningscripts/kubeletstart.ps1

```diff
@@ -66,6 +76,22 @@ if ($global:ContainerRuntime -eq "containerd") {
     $KubeletArgList += @("--container-runtime=remote", "--container-runtime-endpoint=npipe://./pipe/containerd-containerd")
 }
 
+# Update args to get both IPs from NIC using the default route 0.0.0.0/0 if IsDualStackEnabled
+# and the --node-ip argument is not present in the existing kubelet arg list.
+if ($global:IsDualStackEnabled -and !(Test-NodeIPArgExists $KubeletArgList)) {
```

Comment:
When will `Test-NodeIPArgExists $KubeletArgList` return `true`?

Reply:
If --node-ip was already set in the kubeletconfig. We don't do it now, but it's just taking a more cautious approach instead of overwriting the value if it does exist.